examples/src/examples/graphics/particles-random-sprites.mjs

```diff
@@ -4,13 +4,41 @@ import * as pc from 'playcanvas';
  * @param {import('../../options.mjs').ExampleOptions} options - The example options.
  * @returns {Promise<pc.AppBase>} The example application.
  */
-async function example({ canvas, assetPath }) {
+async function example({ canvas, assetPath, deviceType, glslangPath, twgslPath }) {
 
-    const app = new pc.Application(canvas, {
-        mouse: new pc.Mouse(document.body),
-        touch: new pc.TouchDevice(document.body),
-        elementInput: new pc.ElementInput(canvas)
-    });
+    const assets = {
+        'particlesCoinsTexture': new pc.Asset('particlesCoinsTexture', 'texture', { url: assetPath + 'textures/particles-coins.png' }),
+        'particlesBonusTexture': new pc.Asset('particlesBonusTexture', 'texture', { url: assetPath + 'textures/particles-bonus.png' })
+    };
+
+    const gfxOptions = {
+        deviceTypes: [deviceType],
+        glslangUrl: glslangPath + 'glslang.js',
+        twgslUrl: twgslPath + 'twgsl.js'
+    };
+
+    const device = await pc.createGraphicsDevice(canvas, gfxOptions);
+    const createOptions = new pc.AppOptions();
+    createOptions.graphicsDevice = device;
+    createOptions.mouse = new pc.Mouse(document.body);
```

Comment:
I'd remove mouse / touch / element input, it does not seem the example needs any input.